101-data-factory-blob-to-sql-copy/azuredeploy.json

```diff
@@ -0,0 +1,222 @@
+  {
+    "contentVersion": "1.0.0.0",
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#", 
+    "parameters": {
+      "dataFactoryName": {
```

Comment:
Name has to be globally unique. Customers will have to change the datafactoryName parameter before they deploy. Looks like the uniqueString function does not generate a GLBOALLY unique string.